src/SpanInterface.php

```diff
@@ -44,6 +48,14 @@ public function start(): SpanInterface;
      */
     public function activate(): SpanInterface;
 
+    /**
+     * Set the status of this span to one of self::STATUS_*
+     * Description is optional, and MAY be used to describe an error status.
+     *
+     * @return $this
+     */
+    public function setStatus(int $status, ?string $description): SpanInterface;
```

Comment:
Would it be out of place to add an annotation to guide autocomplete?

https://blog.jetbrains.com/phpstorm/2020/12/phpstorm-2020-3-release/#expectedvalues

Reply:
Tried it, wasn't working well